test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1154,6 +1154,49 @@ def test_xnnpack_quantizer_obs_sharing_ops(self):
         ]
         self._test_quantizer(m, example_inputs, quantizer, node_occurrence, node_list)
 
+    def test_xnnpack_quantizer_set_module_name(self):
+        class Sub(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(5, 5)
+
+            def forward(self, x):
+                return self.linear(x)
+
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(5, 5)
+                self.sub = Sub()
+
+            def forward(self, x):
+                x = self.linear(x)
+                x = self.sub(x)
+                return x
+
+        m = M().eval()
+        example_inputs = (torch.randn(3, 5),)
+        quantizer = XNNPACKQuantizer()
+        operator_config = get_symmetric_quantization_config(is_per_channel=True)
```

Comment:
This technically returns a QuantizationConfig, not an OperatorConfig right? Maybe we should rename this variable

Reply:
oh OK

Reply:
maybe let me do a follow up renaming PR